cmd/initializer/dataset/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.11-alpine
+FROM python:3.11-slim-bookworm
```

Comment:
@Debabrata47 Please can you try to build this image locally, and verify that it can download dataset from HF since we don't have e2e tests in our test infra yet?